ghu_web/ghu_global/static/ghu_global/css/style.css

```diff
@@ -44,18 +44,24 @@ nav li {
 }
 
 nav a {
-    color: #555;
+    color: #668cff;
+    background-color: #e6ecff;
     display: inline-block;
     text-align: center;
+    font-size: 20px;
     padding: 8px;
-    border-bottom: 4px solid #aaa;
+    border-bottom: 4px solid #668cff;
 }
 
 nav a:hover {
     color: white;
-    background-color: #aaa;
```

Comment:
Oh, yeah that's caused by Bootstrap.  It gets fixed when the nav's class is set to "navbar".